engine/model.py

```diff
@@ -324,19 +326,34 @@ def set_player_facing(self, facing_x: int, facing_y: int) -> None:
         self.player_sprite.facing_y = facing_y
 
     def activate(self) -> None:
-        """Activates whatever is in front of the player."""
+        """
+        Activates whatever is in front of the player,
+        by checking for collision with a hitbox_sprite.
```

Comment:
No need to update the docstring. In general a docstring describes _what_ the code does, not _how_ it does it. Describing the algorithm violates [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), which can be problematic if we change the algorithm but forget to update the docstring.